File: php/src/Advertisements/User/Application/Command/SignUpMember/SignUpMemberCommand.php
<?php
declare(strict_types=1);

namespace Demo\App\Advertisements\User\Application\Command\SignUpMember;

use SensitiveParameter;

readonly class SignUpMemberCommand
{
    public function __construct(
        public string $securityUserId,
        public string $securityUserRole,
        public string $memberId,
        public string $email,
        #[SensitiveParameter]
        public string $password,
        public string $memberNumber,
        public string $civicCenterId,
    ){}
}

File: kotlin/src/main/kotlin/advertisements/user/application/command/signupmember/SignUpMemberCommand.kt
package advertisements.user.application.command.signupmember

data class SignUpMemberCommand (
    val securityUserId: String,
    val securityUserRole: String,
    val memberId: String,
    val email: String,
    val password: String,
    val memberNumber: String,
    val civicCenterId: String,
)
